app/Services/Twilio.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ivan
+ * Date: 04.06.16
+ * Time: 15:24
+ */
+
+namespace App\Services;
+
+use App\Models\Countries;
+use App\Models\PhoneNumbers;
+use Services_Twilio;
+use Exception;
+use App\Exceptions\NoTwilioAvailableNumberException;
+use App\Exceptions\NoTwilioIncomingNumberException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class Twilio
+{
+
+    const THANKING_MESSAGE_DURATION = 120; //60 * 2 seconds
+
+    const DEFAULT_APPLICATION_SID = 'AP6452316e22874c8e9f4c3425832b0b28';
+
+    const SHORT_CONVERSATION_MESSAGE = 'Thank you very much';
```

Comment:
oh, man please... don't put messages to constants

suggest another approach

Reply:
Found this diff by accident. I have no idea, why some people are so toxic. I'm reading this comment 6 years after and I'm so upset that this is something I was reading in the begging of my career. 

Of course, I can admit that this code is not perfect and should be improved. But isn't it more beneficial for everyone to politely explain what's wrong with it? Beneficial for me, for you and for the entire industry at the end of the day?

I hope, that it was done with a good intention, and you are trying to phrase you thoughts more politely. And I hope, that you agree that it's ok to make mistakes, it's okay to ask any questions, because there is no such thing as a stupid question, but there is no excuse to talk to any people in such a manner.

Reply:
After years I can confirm, that intentions were good.  But comments do not reflect the intention precisely. Definitely, disrespect could be detected here. And I should pick words properly. My apologies.